### PR TITLE
Fix sample project dependencies

### PR DESCRIPTION
**Pull request by dthaler**
Bug was that if you cleaned everything and then tried to build just the
sample_ebpf_ext project, building would fail.  A race condition also
existed when trying to build everything in that if
execution_context_kernel wasn't done building by the
time sample_ebpf_ext was ready to link, the build would also fail.

Signed-off-by: Dave Thaler <dthaler@microsoft.com>

### CODE REVIEW

File: tests/sample/ext/drv/sample_ext.vcxproj [hunk 1]
```diff
@@ -130,11 +130,17 @@
     <ClInclude Include="..\inc\sample_ext_helpers.h" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="$(SolutionDir)libs\execution_context\kernel\execution_context_kernel.vcxproj">
+      <Project>{26e7ed0b-c128-4d7c-a90e-c246def40ad3}</Project>
+    </ProjectReference>
     <ProjectReference Include="$(SolutionDir)libs\platform\kernel\platform_kernel.vcxproj">
       <Project>{fc3f9998-4085-4767-8386-5453f07c3aad}</Project>
     </ProjectReference>
+    <ProjectReference Include="$(SolutionDir)libs\ubpf\kernel\ubpf_kernel.vcxproj">
```
Comment (Alan-Jowett) [inline tests/sample/ext/drv/sample_ext.vcxproj:139]: Why is this even needed?

Comment (dthaler) [reply tests/sample/ext/drv/sample_ext.vcxproj]: Removed.  It was in line 114 before, but it seems to be fine without it.

File: tests/sample/ext/drv/sample_ext.vcxproj [hunk 2]
```diff
@@ -111,7 +111,7 @@
       <RuntimeLibrary>MultiThreadedDebugDll</RuntimeLibrary>
     </ClCompile>
     <Link>
-      <AdditionalDependencies>%(AdditionalDependencies);$(DDK_LIB_PATH)\ntoskrnl.lib;$(DDK_LIB_PATH)\ndis.lib;$(DDK_LIB_PATH)\wdmsec.lib;$(DDK_LIB_PATH)\fwpkclnt.lib;$(SDK_LIB_PATH)\uuid.lib;$(DDK_LIB_PATH)\netio.lib;execution_context_kernel.lib;ubpf_kernel.lib</AdditionalDependencies>
```
Comment (shankarseal) [inline tests/sample/ext/drv/sample_ext.vcxproj:114]: Can you please fix the 2 other vcxproj that has ec and ubpf libs in additionaldependencies section?

findstr -snip execution_context_kernel.lib *.vcxproj
ebpfcore\**EbpfCore.vcxproj**:123:      <AdditionalDependencies>%(AdditionalDependencies);$(DDK_LIB_PATH)\ntoskrnl.lib;$(DDK_LIB_PATH)\ndis.lib;$(DDK_LIB_PATH)\wdmsec.lib;$(DDK_LIB_PATH)\fwpkclnt.lib;$(SDK_LIB_PATH)\uuid.lib;$(DDK_LIB_PATH)\netio.lib;**execution_context_kernel.lib;ubpf_kernel.lib**</AdditionalDependencies>
netebpfext\**netebpfext.vcxproj**:123:      <AdditionalDependencies>%(AdditionalDependencies);$(DDK_LIB_PATH)\ntoskrnl.lib;$(DDK_LIB_PATH)\ndis.lib;$(DDK_LIB_PATH)\wdmsec.lib;$(DDK_LIB_PATH)\fwpkclnt.lib;$(SDK_LIB_PATH)\uuid.lib;$(DDK_LIB_PATH)\netio.lib;**execution_context_kernel.lib;ubpf_kernel.lib**</AdditionalDependencies>
tests\sample\ext\drv\sample_ext.vcxproj:114:      <AdditionalDependencies>%(AdditionalDependencies);$(DDK_LIB_PATH)\ntoskrnl.lib;$(DDK_LIB_PATH)\ndis.lib;$(DDK_LIB_PATH)\wdmsec.lib;$(DDK_LIB_PATH)\fwpkclnt.lib;$(SDK_LIB_PATH)\uuid.lib;$(DDK_LIB_PATH)\netio.lib;execution_context_kernel.lib;ubpf_kernel.lib</AdditionalDependencies>

Comment (dthaler) [reply tests/sample/ext/drv/sample_ext.vcxproj:114]: Done

**Review by shankarseal — APPROVED** [review]

**Review by Alan-Jowett — APPROVED** [review]

**Pull request merged**